### ISSUE TITLE
Remove plugin from plugin manager dialog when it is uninstalled (Trac #118)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#122]
This gives the user feedback that the uninstall was successful.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/118
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "This gives the user feedback that the uninstall was successful.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Remove plugin from plugin manager dialog when it is uninstalled", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-08-01T16:37:06", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120801T16:39:31: 
* ryan@fobel.net commented:

> commit 5f9ee5fb20738fcddd2ecd8ee088528499dc0f32
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Wed Aug 1 12:35:41 2012 -0400
> 
>     Remove plugins from Plugin Manager Dialog once they've been uninstalled
>     
>     Fixes #122


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
